ethcore/src/block.rs

```diff
@@ -137,6 +142,7 @@ impl ExecutedBlock {
 			transactions_set: Default::default(),
 			state: state,
 			traces: if tracing {Some(Vec::new())} else {None},
+			tracing_enabled: tracing,
```

Comment:
You could just add a member function that would return the flag instead of introducing another state variable.